src/react-apps/applications/service-development/src/actions/navigationActions/actions/toggleDrawer/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../navigationActionTypes';
+
+export interface IToggleDrawerActionRejected extends Action {
+  error: Error;
+}
+
+export function toggleDrawerAction(): Action {
+  return {
+    type: ActionTypes.TOGGLE_DRAWER,
+  };
+}
+
```

Comment:
Where is the `toggleDrawerActionFulfilled` action?

Reply:
Removed both fulfilled and rejected action